content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx

```diff
@@ -51,6 +51,7 @@ const ContextTitle = styled(Space).attrs<{ level: number }>(props => ({
   className: font('wb', 3),
   v: { size: 'm', properties: ['margin-bottom'] },
 }))<{ level: number }>``;
+console.log(ContextTitle, 'this is the output of the styling');
```

Comment:
Do you want to keep this?

Reply:
oops! Rookie error, get out of here console log